test/com/jambit/DatabaseConnectionTest.java

```diff
@@ -0,0 +1,70 @@
+package com.jambit;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.jambit.domain.MoodEntries;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+
+/*
+TODO:
+   - More Tests
+   - Delete Database content after all tests (@Afterall)
+   - Implement H2 (optional)
+ */
+
+class DatabaseConnectionTest {
+  private static DatabaseConnection databaseConnection;
+
+  @BeforeAll
+  static void init() throws IOException, SQLException {
+    DatabaseConnection.changePropertyPath("config/appTest.properties");
+    databaseConnection = DatabaseConnection.getInstance();
+  }
+
+  @Test
+  @Order(1)
+  public void insertTest() throws SQLException {
+    MoodEntries testValue =
+        new MoodEntries(0, (int) Math.round(Math.random() * 10), System.currentTimeMillis() / 1000);
+    databaseConnection.writeMoodEntries(testValue);
+    ArrayList<MoodEntries> dbOutputValues = databaseConnection.fetchAllMoodEntries();
+    assertEquals(dbOutputValues.get(dbOutputValues.size() - 1).time, testValue.time);
```

Comment:
* Then check size of return list
* Check expected.equals(actual) for each element. In this case it's only one.